yaml/degoogle.yml

```diff
@@ -6,7 +6,7 @@ web based products:
       eyes: 5
       text: >
         En iyi gizlilik-tabanlı arama motorlarından biri. Alan adının Amerika Birleşik Devletleri'nde kayıtlı 
-        olması bazı kişiler için sıkıntı olabilir.
+        olması bazı kişiler için sıkıntı olabilir. Not: DuckDuckGo 2013 yılında Gizlilik Politikası ile çelişen bir şekilde insanların bilgisayarlarına bir çerez [yerleştirmesi](https://archive.is/qntuk) ile bir dönem gündeme gelmişti. Tam gizlilik için açık kaynak kodlu alternatiflere yönelmenizi öneririz.
```

Comment:
@crazed-translator

Reply:
sizin attığınız linkteki yorumlarda çerezlerin artık konulmadığını yazıyor ama?

Reply:
Şu anda yerleştirmiyor. "Bir dönem" diye bu yüzden yazdım.

Reply:
o zaman listeye yazmamıza gerek görmüyorum, çünkü yazacak olursak diğer başlıklar için de yazmamız gerekecek. en azından genel tercüme bittikten sonra eklemeyi düşünebiliriz.